server/services/itemManager.js

```diff
@@ -19,14 +18,25 @@ class ItemManager {
 		return items;
 	};
 
+	// getItems = async () => {
+	// 	const items = await Items.findAll();
+	// 	return items.map((item) => {
+	// 		return {
+	// 			id: item.id,
+	// 			name: item.itemName,
+	// 			status: item.status,
+	// 		};
+	// 	});
+	// };
```

Comment:
You can delete commented code. no need to leave it in files - it will exist in git history which you can always look at and revert / copy back